docs/release_docs/AHA_Platform_Deployment_Workflows.md

```diff
@@ -0,0 +1,71 @@
+mode of trigger: ```On-Demand```
+
+# AHA Platform Deployment Workflows
+
+## AHA-PROD-CI-CD
+
+This workflow uses two jobs: GitHub-ECR-Tag-Check and Deploy ECS to verify and deploy ```reancare-service``` release to ```aha-prod``` environment
+
+### JOBS
```

Comment:
The Jobs description should come after the below workflow diagrams, like in the REAN Platform doc. Both the docs should be consistent from formatting perspective